add-ons/istio/README.adoc

```diff
@@ -0,0 +1,62 @@
+:linkattrs:
+
+= Istio Add-on
+
+An addon that will deploy https://Istio.io/[Istio] service mesh on to OpenShift provided by Minishift.
+
+Verify that you have installed the add-on by following the https://github.com/minishift/minishift-addons[Minishift add-ons]
+
+== Minishift profile setup
+
+As https://Istio.io/[Istio] setup spins up multiple pods as part of the setup,  hence its ideal to have a 
+setup a minishift profile with the following configuration to avoid any resource crunch.
+
+[code,sh]
+----
+$ minishift profile set servicemesh <1>
+$ minishift config set memory 8GB <2>
+$ minishift config set cpus 3 <3>
+$ minishift config set openshift-version v3.7.0 <4>
+$ minishift addon enable admin-user <5>
+$ minishift addon enable anyuid <6>
+$ minishift start <7>
+----
+
+<1> Create a new Minishift profile in the command its called servicemesh
+<2> Set a memory of 8GB, this is good amount of memory for istio and apps that you might deploy later
+<3> Since https://Istio.io/[Istio] minimum deploys approximately __9__ pods, 3 cpus will give a max of 30 pods
+<4> Since https://Istio.io/[Istio] 0.2.x and above can be deployed with Kubernetes 1.7.x, its required that OpenShift version 3.7.x or above
+<5> Admin user add-on allows you to login with user admin/admin, who has admin privileges on all openshift projects
```

Comment:
Is admin/admin a user, or do you mean system:admin?

Either way, we need to refer to the `admin-user` add-on with backticks and with its name as an add-on. Also, "openshift" needs to be properly capitalized as "OpenShift".